src/components/tasks/CreateButton.jsx

```diff
@@ -0,0 +1,22 @@
+import { ADD_ICON } from "src/utils/constants/constants";
+import "src/components/tasks/CreateButton.scss";
+import PropTypes from "prop-types";
+
+const CreateButton = ({ show, onShow }) => {
+  const handleButtonClick = () => {
+    onShow(!show);
+  };
```

Comment:
please follow name function.
```suggestion
  function handleButtonClick() {
    onShow(!show);
  };
```